<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help::Edit::Patch</class>
 <widget class="QWidget" name="Help::Edit::Patch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="meetaTagGroup">
     <property name="title">
      <string>Meta Tags</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="metaTagEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="messageGroup">
     <property name="title">
      <string>Add Standard Messages</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="messageBangButton">
        <property name="text">
         <string>bang</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="messageIntButton">
        <property name="text">
         <string>int</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="messageFloatButton">
        <property name="text">
         <string>float</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="messageListButton">
        <property name="text">
         <string>list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="messageSignalButton">
        <property name="text">
         <string>signal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="digestGroup">
     <property name="title">
      <string>Digest / Description</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="digestEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="descrptionEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="seeAlsoGroup">
     <property name="title">
      <string>See Also</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="seeAlsoEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
